config.json

```diff
@@ -61,759 +61,759 @@
         "artifactId": "play-services-base",
         "version": "18.2.0",
-        "nugetVersion": "118.2.0.2",
+        "nugetVersion": "118.2.0.3",
         "nugetId": "Xamarin.GooglePlayServices.Base",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-basement",
         "version": "18.2.0",
-        "nugetVersion": "118.2.0.2",
+        "nugetVersion": "118.2.0.3",
         "nugetId": "Xamarin.GooglePlayServices.Basement",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-cast",
         "version": "21.3.0",
-        "nugetVersion": "121.3.0.2",
+        "nugetVersion": "121.3.0.3",
         "nugetId": "Xamarin.GooglePlayServices.Cast",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-cast-framework",
         "version": "21.3.0",
-        "nugetVersion": "121.3.0.2",
+        "nugetVersion": "121.3.0.3",
         "nugetId": "Xamarin.GooglePlayServices.Cast.Framework",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-cast-tv",
         "version": "21.0.0",
-        "nugetVersion": "121.0.0.2",
+        "nugetVersion": "121.0.0.3",
         "nugetId": "Xamarin.GooglePlayServices.Cast.TV",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-clearcut",
         "version": "17.0.0",
-        "nugetVersion": "117.0.0.10",
+        "nugetVersion": "117.0.0.11",
         "nugetId": "Xamarin.GooglePlayServices.Clearcut",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-cloud-messaging",
         "version": "17.0.2",
-        "nugetVersion": "117.0.2.5",
+        "nugetVersion": "117.0.2.6",
         "nugetId": "Xamarin.GooglePlayServices.CloudMessaging",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-code-scanner",
         "version": "16.1.0",
-        "nugetVersion": "116.1.0",
+        "nugetVersion": "116.1.0.1",
         "nugetId": "Xamarin.GooglePlayServices.Code.Scanner",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-cronet",
         "version": "18.0.1",
-        "nugetVersion": "118.0.1.5",
+        "nugetVersion": "118.0.1.6",
         "nugetId": "Xamarin.GooglePlayServices.CroNet",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-deviceperformance",
         "version": "16.0.0",
-        "nugetVersion": "116.0.1.2",
+        "nugetVersion": "116.0.0.2",
```

Comment:
Last/current version was published 3 months ago

https://www.nuget.org/packages/Xamarin.GooglePlayServices.DevicePerformance

116.0.1.2
116.0.1.1
116.0.1

I will fix that just in case and tag for publishing.